index.html

```diff
@@ -1636,8 +1636,18 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="caption">
 			<rdef>caption</rdef>
 			<div class="role-description">
-				<p>On-screen descriptive text for a figure or table in the page.</p>
-				<p>Authors SHOULD reference the element with role <code>caption</code> by setting <pref>aria-describedby</pref> on the figure or table.</p>
+				<p>A visible name or caption for an image, table or grid roles.  See synonym <rref>legend</rref>.</p>
```

Comment:
`caption` *would not* provide a name to an image. This should say `figure`.

Reply:
@scottaohara The reference to an image has been removed in the latest draft